opm/input/eclipse/Deck/FileDeck.cpp

```diff
@@ -521,10 +537,10 @@ void FileDeck::dump_shared(std::ostream& stream,
             // Should ideally use fs::relative()
             std::string include_file = fs::proximate(block.fname, output_dir).generic_string();
             if (include_file.find(block.fname) == std::string::npos) {
-                INCLUDE(stream, include_file);
+                block.is_binary ? IMPORT(stream, include_file) : INCLUDE(stream, include_file);
             }
             else {
-                INCLUDE(stream, block.fname);
+                block.is_binary ? IMPORT(stream, block.fname) : INCLUDE(stream, block.fname);
```

Comment:
Maybe we could refactor this now to avoid the duplicate logic?  Perhaps something along the lines of
```C++
const auto& file_name = (include_file.find(block.fname) == std::string::npos)
    ? include_file : block.fname;
const auto output_func = block.is_binary ? &IMPORT : &INCLUDE;

(*output_func)(stream, file_name);
```